src/components/flow-builder/hooks/useDeleteClick.ts

```diff
@@ -0,0 +1,61 @@
+import { useReactFlow, getConnectedEdges } from "reactflow";
```

Comment:
file name is too generic while this hook could only be used to delete Nodes from flow builder

Reply:
What do you think about naming it **useDeleteNode** instead?